src/main/java/Application/Utilities/EdpHeartbeat.java

```diff
@@ -0,0 +1,44 @@
+package Application.Utilities;
+
+import java.io.IOException;
+import java.net.ConnectException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.time.Duration;
+
+public class EdpHeartbeat {
```

Comment:
What is "Edp"?

Reply:
EDP is Edge Diagnostic Platform, it's a placeholder name so can be changed if needed

Reply:
no problem